### PR TITLE
human bots walk when low on stamina

### PR DESCRIPTION
**Pull request by sweet235**
Because otherwise, they often get stuck in their own base, running against a building until they regain enough stamina to jump. This is especially bad for battlesuits. Do not make them walk while in combat, though.

### CODE REVIEW

**Comment by DolceTriade** [comment]
seems reasonable lgtm

File: src/sgame/sg_bot_util.cpp [hunk 1]
```diff
@@ -2545,6 +2545,19 @@ void BotCalculateStuckTime( gentity_t *self )
 	self->botMind->lastThink = level.time;
 }
 
+static Cvar::Cvar<int> g_bot_walkAfterCombat( "g_bot_walkAfterCombat", "how many milliseconds must pass after combat until human bots are allowed to walk when low on stamina", Cvar::NONE, 3000 );
+
```
Comment (DolceTriade) [inline src/sgame/sg_bot_util.cpp:2549]: Is this cvar actually helpful?

Comment (sweet235) [reply src/sgame/sg_bot_util.cpp:2549]: No idea.

Comment (sweet235) [reply src/sgame/sg_bot_util.cpp]: I NUKED it.

**Pull request merged**